pennylane/templates/subroutines/qsvt.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Contains the QSVT template and qsvt wrapper function.
+"""
+import numpy as np
+import pennylane as qml
+from pennylane.queuing import QueuingManager
+from pennylane.ops import BlockEncode, PCPhase
+from pennylane.ops.op_math import adjoint
+from pennylane.operation import AnyWires, Operation
+
+
+def qsvt(A, phi_vect, wires):
```

Comment:
Might consider a different name instead of `phi_vect`, e.g., just `angles`.

Reply:
Changed to `angles`